src/test/test_repo_tester.py

```diff
@@ -0,0 +1,22 @@
+import unittest                                                                 
+from main.repo_tester import *                                                  
+                                                                                
+class test_repo_tester(unittest.TestCase):                                      
+                                                                                
+    """                                                                         
+    Should, given a pull request type event, clone the repo,                    
+    switch to the correct branch, run "ci.sh" in the root                       
+    of the repository, remove the repo, and finally return                      
+    the exit code of "ci.sh". for json_pull_request_example.txt
+    the exit code should be 10.
+    """                                                                         
+    def test_1(self):                                                           
+        with open('test/json_pull_request_example.txt', 'r') as payloadfile:         
+            payload = payloadfile.read()                                        
+            self.assertTrue(repo_test(payload) == 10)                           
```

Comment:
Why should the exit code be 10? In the doc it says that it is either 0, 1 or 2

Reply:
The comment says the exit code should be 0, 1 or 2, but in reality the function will return whatever exit code ci.sh returns. In my demo repo the contents of ci.sh is:

```
echo "hello"
exit 10
```

A proper ci.sh should return 0, 1 or 2.

Reply:
Can we ensure that the tests return that?

Reply:
I'm 95% sure that python unittests return exit status 1 if a test fail, and 0 if all tests succeed.
I'm 99% sure that python static syntax checking returns 1 if it fails and 0 if it succeeds. The ci.sh should be very short and simple.